data/data-pipeline/data_pipeline/etl/sources/tree_equity_score/etl.py

```diff
@@ -87,4 +87,4 @@ def load(self) -> None:
         logger.info("Saving Tree Equity Score GeoJSON")
         # write nationwide csv
         self.CSV_PATH.mkdir(parents=True, exist_ok=True)
-        self.df.to_file(self.CSV_PATH / "tes_conus.geojson", driver="GeoJSON")
+        self.df.to_csv(self.CSV_PATH / "tes_conus.csv", index=False)
```

Comment:
For consistency with other ETL classes, I'd say let's change the whole file name to `usa.csv` instead of `tes_conus.csv`

Reply:
Got it - thanks @switzersc-usds